pom.xml

```diff
@@ -374,6 +439,8 @@
                   <transformers>
                     <!-- Hadoop uses service loader to find filesystem impls, without this may not find them -->
                     <transformer implementation="org.apache.maven.plugins.shade.resource.ServicesResourceTransformer" />
+                    <!-- New transformer. Stops "unrecognized format specifier" error. -->
```

Comment:
Let's add a comment with a link to `https://github.com/edwgiz/maven-shaded-log4j-transformer` in case anybody has questions about what this does. The README there should answer any additional questions.